addons/binding/org.openhab.binding.meterreader/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,43 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: MeterReader Binding
+Bundle-SymbolicName: org.openhab.binding.meterreader;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.3.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-ClassPath: .,
+ lib/jsml.jar,
+ lib/j62056.jar
+Import-Package: 
+ gnu.io,
+ javax.measure;version="1.0.0",
+ javax.measure.format;version="1.0.0",
+ javax.measure.quantity;version="1.0.0",
+ javax.measure.spi;version="1.0.0",
+ org.apache.commons.codec;version="1.6.0",
+ org.apache.commons.codec.binary;version="1.6.0",
+ org.apache.commons.lang;version="2.6.0",
+ org.apache.commons.net;version="3.2.0",
+ org.apache.commons.net.telnet;version="3.2.0",
+ org.eclipse.jdt.annotation;resolution:=optional,
+ org.eclipse.smarthome.config.core,
+ org.eclipse.smarthome.core.common,
+ org.eclipse.smarthome.core.library.types,
+ org.eclipse.smarthome.core.library.unit,
+ org.eclipse.smarthome.core.thing,
+ org.eclipse.smarthome.core.thing.binding,
+ org.eclipse.smarthome.core.thing.binding.builder,
+ org.eclipse.smarthome.core.thing.type,
+ org.eclipse.smarthome.core.types,
+ org.hamcrest;version="1.1.0";resolution:=optional,
+ org.junit;version="4.11.0";resolution:=optional,
+ org.openhab.binding.meterreader,
+ org.osgi.service.component;version="1.2.2",
+ org.slf4j,
+ tec.uom.se.spi;version="1.0.8"
```

Comment:
You must not depend on tec.uom.

Reply:
I could not find out how to add new units to the system without depending on the implementation. In my case I want to add the Unit "kWh" (*SmartHomeUnits.WATT.multiply(SmartHomeUnits.HOUR)*).

Reply:
Hm, there should be no need to define a new unit (or rather missing stuff should be added to SmartHomeUnits directly). Especially "kWh" I very much expect to be supported by ESH, but there seems to be indeed be a problem with it (I have just created https://github.com/eclipse/smarthome/issues/5341 for it). @htreu, please comment on how this can be solved.